templates/assistants/assistant_form.html

```diff
@@ -16,6 +16,21 @@
   {% if object.id %}
     <h2 class="text-l pg-text-muted">Assistant ID: {{ object.assistant_id }}</h2>
   {% endif %}
+  <div id="sync-wrapper" class="flex w-full gap-2"
+       hx-get="{% url 'assistants:check_sync_status' request.team.slug object.pk %}"
```

Comment:
This would need to be done only when `object` exists. For new assistants, object will be `None`

Reply:
you could wrap the whole 'sync-wrapper' div in an if statement `{% if object.pk %}`

Reply:
good call https://github.com/dimagi/open-chat-studio/pull/774/commits/28567c77c22303ff4725d5de803a462f33f72798